source/common/stats/symbol_table_creator.cc

```diff
@@ -4,7 +4,7 @@ namespace Envoy {
 namespace Stats {
 
 bool SymbolTableCreator::initialized_ = false;
-bool SymbolTableCreator::use_fake_symbol_tables_ = true;
+bool SymbolTableCreator::use_fake_symbol_tables_ = false;
```

Comment:
Once everything else in the PR looks good, this 'false' should be flipped back to 'true' before a final round of CI/approval/merge.

Reply:
You can also revert the other warning in options_impl also, right?

Reply:
Yes this is fine I didn't see the revert in this PR when I wrote this.